src/Http/Response.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * This code is licensed under the BSD 3-Clause License.
+ *
+ * Copyright (c) 2017, Maks Rafalko
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its
+ *   contributors may be used to endorse or promote products derived from
+ *   this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Http;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @internal
+ */
+final class Response
+{
+    public const CREATED_RESPONSE_CODE = 201;
+
+    private $body;
+    private $statusCode;
+
+    public function __construct(string $body, int $statusCode)
+    {
+        Assert::range($statusCode, 200, 599);
+
+        $this->body = $body;
+        $this->statusCode = $statusCode;
+    }
+
+    public function getBody(): string
```

Comment:
I'm not a big fan of omitting it: 
- it can create some confusion at whether you mean the property or method
- it is the convention so far in the codebase
- using a public property here would mean loosing type and making this VO mutable

Reply:
>I'm not a big fan of omitting it:
>
>* it can create some confusion at whether you mean the property or method

We have static code analysis for this.

>* it is the convention so far in the codebase

There are classes that do not have the `get` prefixes for getters, for example, [`BuildContext`](https://github.com/infection/infection/blob/5f3094ca95593454f47382992ee7ac4d78463c9b/src/Environment/BuildContext.php#L41-L60).

Reply:
Which is pretty much the exception, brought by you, rather than the norm :)

>We have static code analysis for this.

I don't want to require an SA to read. It's minor granted, but it's still a mental overhead which is unnecessary to me since it does not make it more expressive IMO.